modules/tinymce/src/themes/silver/main/ts/ui/window/SilverInlineDialog.ts

```diff
@@ -24,11 +24,23 @@ interface RenderedDialog<T extends Dialog.DialogData> {
   readonly instanceApi: Dialog.DialogInstanceApi<T>;
 }
 
+const getDialogSizeClasses = (size: Dialog.DialogSize): string[] => {
```

Comment:
Could you name this getInlineDialogSizeClasses since this is specific for inline dialogs. Since the render function is named inline so things are kind of inline name spaced. It would also show that the function is different to the very similar function that is in the SilverDialog module. This is internal so no biggie, feel free to ignore.